Makefile

```diff
@@ -145,7 +146,7 @@ standalone: node_modules
 .PHONY: add-on
 add-on: node_modules
 	mkdir -p $(BUILT)/$@
-	$(RSYNC) $@/chrome.manifest $@/chrome/bootstrap.js $(BUILT)/$@
+	$(RSYNC) $@/jar.mn $@/chrome/bootstrap.js $(BUILT)/$@
```

Comment:
We don't need the jar.mn here - just remove it, then add `$(BUILT)/$(ADD-ON)/chrome.manifest` as a dependency on the `add-on:` line above.